Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-mirroring.md

```diff
@@ -0,0 +1,166 @@
+
+## Create the Filesystem with Mirroring enabled
+
+The following will enable mirroring on the filesystem:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystem
+metadata:
+  name: myfs
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+  dataPools:
+    - name: replicated
+      failureDomain: host
+      replicated:
+        size: 3
+  preserveFilesystemOnDelete: true
+  metadataServer:
+    activeCount: 1
+    activeStandby: true
+  mirroring:
+    enabled: true
+    # list of Kubernetes Secrets containing the peer token
+    # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
+    # Add the secret name if it already exists else specify the empty list here.
+    peers:
+      secretNames:
+        #- secondary-cluster-peer
+    # specify the schedule(s) on which snapshots should be taken
+    # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
+    snapshotSchedules:
+      - path: /
+        interval: 24h # daily snapshots
+        # The startTime should be mentioned in the format YYYY-MM-DDTHH:MM:SS
+        # If startTime is not specified, then by default the start time is considered as midnight UTC.
+        # see usage here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#usage
+        # startTime: 2022-07-15T11:55:00
+    # manage retention policies
+    # see syntax duration here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-retention-policies
+    snapshotRetention:
+      - path: /
+        duration: "h 24"
+```
+
+## Create the cephfs-mirror daemon
+
+On the source storage cluster, deploy the `cephfs-mirror`  daemon by referring to [Filesystem Mirror CRD](../../CRDs/Shared-Filesystem/ceph-fs-mirror-crd.md).
+
+## Configuring mirroring peers
+
+Once mirroring is enabled, Rook will by default create its own [bootstrap peer token](https://docs.ceph.com/en/latest/dev/cephfs-mirroring/?#bootstrap-peers) so that it can be used by another cluster.
+The bootstrap peer token can be found in a Kubernetes Secret. The name of the Secret is present in the Status field of the CephFilesystem CR:
+
+```yaml
+status:
+  info:
+    fsMirrorBootstrapPeerSecretName: fs-peer-token-myfs
+```
+
+This secret can then be fetched like so:
+
+```console
+$ kubectl get secret -n rook-ceph fs-peer-token-myfs -o jsonpath='{.data.token}'|base64 -d
+eyJmc2lkIjoiOTFlYWUwZGQtMDZiMS00ZDJjLTkxZjMtMTMxMWM5ZGYzODJiIiwiY2xpZW50X2lkIjoicmJkLW1pcnJvci1wZWVyIiwia2V5IjoiQVFEN1psOWZ3V1VGRHhBQWdmY0gyZi8xeUhYeGZDUTU5L1N0NEE9PSIsIm1vbl9ob3N0IjoiW3YyOjEwLjEwMS4xOC4yMjM6MzMwMCx2MToxMC4xMDEuMTguMjIzOjY3ODldIn0=
+```
+
+The secret must be decoded. The result will be another base64 encoded blob that you will import in the destination cluster:
+
+```console
+external-cluster-console # ceph fs snapshot mirror peer_bootstrap import <fs_name> <token file path>
```

Comment:
I'd suggest having a section header than indicates what is executed in the external cluster, instead of the console prompt name
```suggestion
ceph fs snapshot mirror peer_bootstrap import <fs_name> <token file path>
```